Formula/a/amass.rb

```diff
@@ -23,9 +23,7 @@ def install
   end
 
   test do
-    output = shell_output("#{bin}/amass intel -whois -d owasp.org 2>&1")
-    assert_match "blockster.com", output
-
+    assert_match "dns.google", shell_output("#{bin}/amass intel -active -addr 8.8.8.8")
```

Comment:
Test sporadically fails so not ideal. May just need to run something other than `amass intel` as we need consistent results in CI (given this will be tested as `go` dependent).